### PR TITLE
style: improve tx flow modal

### PR DESCRIPTION
**Pull request by bonustrack**
Improve the tx flow modal style by reducing the size of the icon

For SX:
image
image
image

For settings change:
image
image

For S:
image

Also fix the ecosystem app page

### CODE REVIEW

File: apps/ui/src/views/App.vue [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <script setup lang="ts">
-import { simplifyURL } from '@/helpers/utils';
+import { getUrl, simplifyURL } from '@/helpers/utils';
```
Comment (wa0x6e) [inline apps/ui/src/views/App.vue:2]: These changes does not seem to be related to this PR

Comment (bonustrack) [reply apps/ui/src/views/App.vue:2]: Right, this is to fix ecosystem app page I should have done another PR
